src/index.js

```diff
@@ -0,0 +1,94 @@
+// eslint-disable-next-line
+import completeTasks from './completeTasks.js';
+import './style.css';
+
+const todoForm = document.querySelector('#todo-form');
+const todoList = document.querySelector('.todos');
+const clearAll = document.querySelector('#completed-tasks');
+const mainInput = document.querySelector('#todo-form input');
+
+// eslint-disable-next-line
+export let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function createTask(task) {
+  const taskEl = document.createElement('li');
+  taskEl.setAttribute('id', task.id);
+  taskEl.classList.add('item');
+  if (task.completed) {
+    taskEl.classList.add('complete');
+  }
+  const innerHTML = `<div class = "list"> <input class = "checkbox" type = "checkbox" id = "${task.id}" ${task.completed ? 'checked' : ''} name = "tasks" class= "taskCompleted">
```

Comment:
- [ ] Kindly consider using `index` instead of `id` when you store tasks, also make sure when a task is deleted the `index` is re-arranged so if there are 3 tasks and we delete the second task the lask task index should change from `3` to `2`

Reply:
Ok, ID is rearranged now.

Reply:
Fixed.